ext/gd/libgd/gd_gif_out.c

```diff
@@ -129,8 +129,7 @@ void gdImageGifCtx(gdImagePtr im, gdIOCtxPtr out)
 static int _gdImageGifCtx(gdImagePtr im, gdIOCtxPtr out)
 {
 	gdImagePtr pim = 0, tim = im;
-	int interlace, BitsPerPixel;
-	interlace = im->interlace;
```

Comment:
This should also be handled [like upstream libgd](https://github.com/libgd/libgd/blob/e0cb1b76c305db68b251fe782faa12da5d357593/src/gd_gif_out.c#L231-L262). Actually, there is a minor bug in the bundled libgd, where truecolor images with the interlace flag set will nonetheless be stored non-interlaced (since `gdImageCreatePaletteFromTrueColor()` doesn't copy the interlace flag). Probably it's sensible to actually treat this issue as bug and to apply the fix (and a regression test) to PHP-7.2, while the rest of this PR should target PHP-7.4.

Reply:
Ok, so I reverted the modification to this file.

Reply:
Fine! The interlace flag bug is fixed with 41fb0eaa116da34fa531437d5b87ca73089e4944.